TYPO3.Flow/Migrations/Mysql/Version20141015125841.php

```diff
@@ -67,6 +70,8 @@ public function postUp(Schema $schema)
                 continue;
             }
 
+            $this->write(print_r(array('persistent', $mediaType, $md5, $filesize, $resourceInfo['persistence_object_identifier']), true));
```

Comment:
This looks like debug output, is that intentionally left here?

Reply:
Can be removed, i used it during the migration.

Reply:
Not creating the table is also possible. One solution would be to check if the migration can be applied before it starts and do not fail at 95%